x-pack/plugins/ml/server/models/job_service/jobs.ts

```diff
@@ -257,14 +257,17 @@ export function jobsProvider(client: IScopedClusterClient, mlClient: MlClient) {
     return { jobs, jobsMap };
   }
 
-  async function createFullJobsList(jobIds: string[] = []) {
+  async function createFullJobsList(jobIds: string[] = [], excludeGenerated = false) {
     const jobs: CombinedJobWithStats[] = [];
     const groups: { [jobId: string]: string[] } = {};
     const datafeeds: { [id: string]: DatafeedWithStats } = {};
     const calendarsByJobId: { [jobId: string]: string[] } = {};
     const globalCalendars: string[] = [];
 
     const jobIdsString = jobIds.join();
+
+    const datafeedParams = excludeGenerated ? { exclude_generated: excludeGenerated } : undefined;
```

Comment:
As above, did you mean to include these changes? Looks like they are part of the `exclude_generated` work.

Reply:
This is now reverted https://github.com/elastic/kibana/pull/88880/commits/311892828ba8ad9bf92efe643c97c43d7c0439a1